config/routes.rb

```diff
@@ -2,10 +2,7 @@
 
   namespace :api do
     namespace :v1 do
-      namespace :trails do
-        get '/', to: "trails#index"
-        get '/:id', to: "trails#show"
-      end
+      resources :trails, defaults: {format: 'json'}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.